_posts/2020-01-26-support-the-52Fund.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Supporting the #52Fund
+date: 2020-01-26 14:16:02
+---
+
+It's January, so it must mean a new post by Digital Preservation is People.
+Whatever shape this project eventually takes, there is a commitment to keep
+communicating the important things that are happening in our community.
+
+Today, we're talking gate-keeping. Or, _how your ideas on what makes a
+good leader is entirely a construct_.
+
+## Background
+
+The way the news surrounding the SAA vice-president election reached me was
+first via:
+
+* Courtney Chartier's blog: [here][saa-1].
+* Ruth Kitchin Tillman's blog: [here][saa-2].
+* Archives for Black Lives in Philadelphia: [here][saa-3]
+
+Which already cover the issue in huge and important detail.
+
+The summary is that 52 members of the SAA have petitioned to place a new
+nomination on the ballot for Vice President/President elect position.
+
+* [Here's that petition][saa-4], and the [SAA update on the procedure][saa-5].
+
+The only context provided (if you go looking for it) is as follows:
+
+> unsatisfied with the lack of a choice of two candidates with significant
+> organizational experience posed by the slate recently put forward for the
+> office of SAA vice-president/president-elect … I feel strongly that SAA
+> members need a choice of candidates who have deep, varied experience working
+> for our profession within our national organization, and multi-faceted
+> connections to the concerns of SAA’s leadership.”
+
+## Why is this a problem?
+
+By-laws aside, 52 signers decided to subvert an open-ballot, open to an entire
+organization to have their say, and put their weight behind someone they
+considered credible.
+
+52 signers sought an end-around of the current Chair of the Nominating
+Committee. Themselves [seeking to][saa-6] do the following:
+
+> I think it will be critical to put a slate of candidates together that will
+> have a strong portfolio of success in making room for historically
+> underrepresented identities in leadership positions, who advocate for success
+> of these communities and are willing to call out and address discriminatory
+> practices within the profession and in spaces supposedly designed to nurture
+> and support emerging leaders and change agents.
```

Comment:
Add an attribution to this quote, like who exactly said it/wrote it and the context